lib/dead_end/core_ext.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
```

Comment:
```suggestion
require_relative "version"

module DeadEnd
  class << self
    def handle_error(e)
      require_relative "api"
      handle_error(e)
    end

    alias handle_error handle_error
  end
end

if ::SyntaxError.method_defined?(:detailed_message)
  module Handler
    def detailed_message(highlight: nil, **)
      if highlight and /\A(?:require(?:_relative)?|load)\z/.match?(backtrace_locations.first.label)
        DeadEnd.handle_error(self)
      end
    rescue SystemStackError => e
      STDERR.puts e
    ensure
      super
    end

    ::SyntaxError.prepend(self)
  end

  return
end

```

Reply:
Updated in https://github.com/zombocom/dead_end/pull/139.